<launch>
  <arg name="role_name" default="hero" />
  <arg name="control_loop_rate" default="0.1" />

  <node pkg="mock" type="mock_traffic_light.py" name="mock_traffic_light" output="screen">
        <param name="control_loop_rate" value="$(arg control_loop_rate)" />
        <param name="role_name" value="$(arg role_name)" />
  </node>

</launch>
